test/oneflow.test.js

```diff
@@ -115,6 +115,33 @@ describe('Oneflow', function () {
 			    'content-type',
 		    ]);
 		});
+		
+		it('should perform an http request with data=null', async () => {
+		    stubs.axiosRequest = sinon.stub(sdk.client, 'request').callsFake(a => Promise.resolve({ data: a }));
+
+		    const result = await sdk.request('get', '/order', null);
+
+		    result.method.should.be.equal('get');
+		    result.data.should.be.eql({});
+		});
+		
+		it('should perform an http request with data=undefined', async () => {
+		    stubs.axiosRequest = sinon.stub(sdk.client, 'request').callsFake(a => Promise.resolve({ data: a }));
+
+		    const result = await sdk.request('get', '/order', undefined);
+
+		    result.method.should.be.equal('get');
+		    result.data.should.be.eql({});
+		});
+		
+		it('should perform an http request with data=""', async () => {
```

Comment:
Well, in this case, this only is valid if the content type is JSON, if you were to post a string, it would be valid.

Reply:
yeah, in that case it's better to redefine only `null`, because undefined is ok
will fix it, thanks